pkgs/applications/window-managers/jwm/default.nix

```diff
@@ -1,22 +1,33 @@
-{ stdenv, fetchurl, libX11, libXext, libXinerama, libXpm, libXft, freetype,
+{ stdenv, fetchurl, pkgconfig, automake, autoconf, libtool, libX11, libXext,
+  libXinerama, libXpm, libXft, which, xorg, libXau, libXdmcp, libpng, libjpeg,
+  expat, xproto, xextproto, xineramaproto, librsvg, gettext, freetype,
   fontconfig }:
 
 stdenv.mkDerivation rec {
-  name = "jwm-2.2.2";
+  name = "jwm-${version}";
+  version = "s1406";
```

Comment:
A version should not start with an alphabetical character. If it does nix won't be able to distinct between a derivations name and its version.